package.json

```diff
@@ -51,7 +51,7 @@
     "@changesets/changelog-github": "0.4.1",
     "husky": ">=6",
     "lint-staged": ">=10",
-    "prettier": "2.3.2",
+    "prettier": "2.4.1",
```

Comment:
This update allows prettier to work when there are react components wrapping markdown content like:

```md
<Example>

Testing

</Example>
```

Reply:
🤔 Out of curiousity, what specific behavior changed between Prettier versions?

Reply:
https://prettier.io/blog/2021/09/09/2.4.0.html

It looks like:
* Typescript 4.4 support
* Replace jsxBracketSameLine option with bracketSameLine option

are the main things